src/modules/shortcut_guide/shortcut_guide.cpp

```diff
@@ -134,6 +157,16 @@ void OverlayWindow::disable(bool trace_event)
         target_state->exit();
         target_state.reset();
         winkey_popup.reset();
+
+        bool success = UnhookWindowsHookEx(hook_handle);

```

Comment:
Wouldn't it be better to check for `hook_handle != nullptr` before calling `UnhookWindowsHookEx(hook_handle)`?